R/drive_publish.R

```diff
@@ -64,13 +74,16 @@ drive_change_publish <- function(file,
       message(
         glue_data(
           file_update,
-          "You have changed the publication status of {sq(name)}."
+          "\nYou have changed the publication status of file:\n",
+          "  * {sq(name)}."
         )
```

Comment:
When we only pull one value out of something like `file_update`, perhaps we should shift to `glue("... {file_update$name}")`.